projects/components/src/table/cells/data-renderers/duration/duration-table-cell-renderer.component.ts

```diff
@@ -0,0 +1,30 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { TimeDuration, TimeUnit, UnitStringType } from '@hypertrace/common';
+import { TableCellRenderer } from '../../table-cell-renderer';
+import { TableCellRendererBase } from '../../table-cell-renderer-base';
+import { CoreTableCellParserType } from '../../types/core-table-cell-parser-type';
+import { CoreTableCellRendererType } from '../../types/core-table-cell-renderer-type';
+import { TableCellAlignmentType } from '../../types/table-cell-alignment-type';
+
+@Component({
+  selector: 'ht-duration-table-cell-renderer',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  // TODO: Modify the htDuration pipe to also support long format display and use it here.
```

Comment:
lets fix it right away. its a small change. Update htDuration pipe to take a format input.

Reply:
updated pipe